app/models/identity.rb

```diff
@@ -2,6 +2,10 @@ class Identity < ActiveRecord::Base
   include NonNullUuid
 
   belongs_to :user
+  belongs_to :sp,
```

Comment:
I'm 👎  on this. If this relationship does not exist in the code, it should not be here. Why would we need to create this relationship in ERD if it does not exist in the app? Currently, an Identity does not belong to a Service Provider. A Service Provider does not have many identities. `service_provider` is a column in the Identities table.

A User can have many identities, but only one identity per service provider, where service_provider is one of many attributes of that identity. That is what the diagram should show. I don't think the diagram should try to link the Service Provider entity with the Identity entity since it does not exist in the app.